package.json

```diff
@@ -225,7 +225,7 @@
     "react-native-branch": "5.3.1",
     "react-native-change-icon": "4.0.0",
     "react-native-circular-progress": "1.3.8",
-    "react-native-cloud-fs": "rainbow-me/react-native-cloud-fs#9b204615b76cf3d29bd86a9094dbd95d717b6a7a",
+    "react-native-cloud-fs": "rainbow-me/react-native-cloud-fs#247c245ff4da8de5bde47b627831798b49087868",
```

Comment:
should we merge this and use the merge commit instead?

Reply:
Yes please!